view/statistics-files-pending.js

```diff
@@ -102,8 +103,20 @@ exports['statistics-main'] = function () {
 			this.processingStepsMeta.frontDesk ?
 					generateRow({ class: 'statistics-table-sub-header statistics-table-sub-header-success' },
 						_("Pending for withdraw at Front Desk"), services, function (data) {
-							data = data.atPartB.getBySKeyPath(resolveFullStepPath('frontDesk'));
-							return data ? data._pending : null;
+							var result = null;
+							if (!getFrontDeskNames().size) return null;
```

Comment:
We should not have that check, instead we should fix condition at 103 (` 		this.processingStepsMeta.frontDesk ?`)